src/Services/Masa.Dcc.Service/Services/ProjectService.cs

```diff
@@ -25,21 +25,21 @@ public async Task<List<ProjectAppsModel>> GetProjectListAsync(string envName)
         return result;
     }
 
-    public async Task<ProjectDetailModel> GetAsync(int Id)
+    public async Task<ProjectDetailModel> GetAsync(int id)
     {
-        var result = await _pmClient.ProjectService.GetAsync(Id);
+        var result = await _pmClient.ProjectService.GetAsync(id);
 
         return result;
     }
 
-    public async Task<List<ProjectModel>> GetListAsync()
+    public async Task<List<BuildingBlocks.StackSdks.Pm.Model.ProjectModel>> GetListAsync()
```

Comment:
BuildingBlocks.StackSdks.Pm.Model.  ?

Reply:
it is confict with BuildingBlocks.StackSdks.Auth.Model.

Reply:
![image](https://user-images.githubusercontent.com/7838147/236722934-5b25bbfa-018e-4159-bf67-2f54f5e47ce0.png)